diff --git a/Sandboxie/core/drv/verify.c b/Sandboxie/core/drv/verify.c
index b61b8c04..858488bd 100644
--- a/Sandboxie/core/drv/verify.c
+++ b/Sandboxie/core/drv/verify.c
@@ -228,6 +228,8 @@ NTSTATUS KphVerifySignature(
     PVOID hash = NULL;
     ULONG hashSize;
 
+    // 此处直接返回
+	return STATUS_SUCCESS;
     // Import the trusted public key.
 
     if (!NT_SUCCESS(status = BCryptOpenAlgorithmProvider(&signAlgHandle, KPH_SIGN_ALGORITHM, NULL, 0)))
@@ -1094,4 +1096,4 @@ void InitFwUuid()
 
         //DbgPrint("sbie FW-UUID: %S\n", g_uuid_str);
     }
-}
\ No newline at end of file
+}
